yogstation/code/game/gamemodes/gangs/gangtool.dm

```diff
@@ -0,0 +1,260 @@
+	addtimer(CALLBACK(src, .proc/recall2, user), rand(100,300))
+
+/obj/item/device/gangtool/proc/recall2(mob/user)
+	if(!recallchecks(user))
+		return
+	to_chat(user, "<span class='info'>[icon2html(src, loc)]Shuttle recall order generated. Accessing station long-range communication arrays...</span>")
+	addtimer(CALLBACK(src, .proc/recall3, user), rand(100,300))
+
+/obj/item/device/gangtool/proc/recall3(mob/user)
+	if(!recallchecks(user))
+		return
+	var/list/living_crew = list()//shamelessly copied from mulligan code, there should be a helper for this
+	for(var/mob/Player in GLOB.mob_list)
+		if(Player.mind && Player.stat != DEAD && !isnewplayer(Player) && !isbrain(Player) && Player.client)
+			living_crew += Player
+	var/malc = CONFIG_GET(number/midround_antag_life_check)
+	if(living_crew.len / GLOB.joined_player_list.len <= malc) //Shuttle cannot be recalled if too many people died
+		to_chat(user, "<span class='warning'>[icon2html(src, user)]Error: Station communication systems compromised. Unable to establish connection.</span>")
+		recalling = FALSE
+		return
+	to_chat(user, "<span class='info'>[icon2html(src, loc)]Comm arrays accessed. Broadcasting recall signal...</span>")
+	addtimer(CALLBACK(src, .proc/recallfinal, user), rand(100,300))
+
+/obj/item/device/gangtool/proc/recallfinal(mob/user)
+	if(!recallchecks(user))
+		return
+	recalling = FALSE
+	log_game("[key_name(user)] has tried to recall the shuttle with a gangtool.")
+	message_admins("[key_name_admin(user)] has tried to recall the shuttle with a gangtool.", 1)
+	if(SSshuttle.cancelEvac(user))
+		gang.recalls--
+		return TRUE
+
+	to_chat(user, "<span class='info'>[icon2html(src, loc)]No response recieved. Emergency shuttle cannot be recalled at this time.</span>")
+	return
+
+/obj/item/device/gangtool/proc/recallchecks(mob/user)
+	if(!can_use(user))
+		return
+	if(SSshuttle.emergencyNoRecall)
+		return
+	if(!gang.recalls)
+		to_chat(user, "<span class='warning'>Error: Unable to access communication arrays. Firewall has logged our signature and is blocking all further attempts.</span>")
+		return
+	if(SSshuttle.emergency.mode != SHUTTLE_CALL) //Shuttle can only be recalled when it's moving to the station
+		to_chat(user, "<span class='warning'>[icon2html(src, user)]Emergency shuttle cannot be recalled at this time.</span>")
+		recalling = FALSE
+		return
+	if(!gang.dom_attempts)
+		to_chat(user, "<span class='warning'>[icon2html(src, user)]Error: Unable to access communication arrays. Firewall has logged our signature and is blocking all further attempts.</span>")
+		recalling = FALSE
+		return
+	if(!is_station_level(user.z)) //Shuttle can only be recalled while on station
+		to_chat(user, "<span class='warning'>[icon2html(src, user)]Error: Device out of range of station communication arrays.</span>")
+		recalling = FALSE
+		return
+	return TRUE
+
+/obj/item/device/gangtool/proc/can_use(mob/living/carbon/human/user)
+	if(!istype(user))
+		return
+	if(user.incapacitated())
```

Comment:
seriously make 100% sure that you can use this while you don't have legs.

Reply:
yep works without legs